mtp_api/assets/core/js/satisfaction-results.js

```diff
@@ -48,27 +53,112 @@ django.jQuery(function($) {
         legendBounds = cli.getBoundingBox('legend'),
-        $title;
+        chartBounds = cli.getChartAreaBoundingBox(),
+        significantWidth = chartBounds.width * 5 / 6, // last option is disregarded
+        meanMarkerWidth = 12,
+        $hideOnMouseover = [],
+        $chartDetails, $title,
+        i;
+
+      $chartDetails = $($svg.children('g')[1]).children('g');
+      $chartDetails = $($chartDetails[0]).add($chartDetails[$chartDetails.length - 1]).attr({
+        visibility: 'hidden'
+      });
+
+      for (i in colourScale) {
+        if (colourScale.hasOwnProperty(i)) {
+          var colour = colourScale[i],
+            barBounds = cli.getBoundingBox('bar#0#' + i),
+            $rect = $(document.createElementNS(svgNamespace, 'rect'));
+          $rect.attr({
+            x: barBounds.left,
+            y: barBounds.top + barBounds.height + 2,
+            width: barBounds.width * 2,
+            height: 3,
+            fill: colour
+          });
+          $svg.append($rect);
+          $hideOnMouseover.push($rect);
+        }
+      }
 
-      $title = $(document.createElementNS($svg[0].namespaceURI, 'text'));
+      for (i in means) {
+        if (means.hasOwnProperty(i)) {
+          var mean = means[i],
+            $meanMarker = $(document.createElementNS(svgNamespace, 'rect')),
+            $meanTextMarker = $(document.createElementNS(svgNamespace, 'text')),
+            x, y;
+          if (mean === null) {
+            x = chartBounds.width * (1 - (1 / 6) + (1 / 12));
+          } else {
+            x = significantWidth * (2 + mean) / 4
+          }
+          x = chartBounds.left + x - meanMarkerWidth / 2;
+          y = chartBounds.top;
+          $meanMarker.attr({
+            x: x,
+            y: y,
+            width: meanMarkerWidth,
+            height: chartBounds.height,
+            fill: colours[i]
+          });
+          x += 1;
+          y = chartBounds.top + chartBounds.height / 2;
+          $meanTextMarker.attr({
+            x: x,
+            y: y - 2,
+            transform: 'rotate(90,' + x + ',' + y + ')',
+            fill: '#fff',
+            'text-anchor': 'middle',
+            'text-align': 'center',
+            'font-family': font,
+            'font-size': 10
+          }).text('Average response');
+          $hideOnMouseover.push($meanMarker);
+          $hideOnMouseover.push($meanTextMarker);
+        }
+      }
+      $svg.append($hideOnMouseover);
+
+      $title = $(document.createElementNS(svgNamespace, 'text'));
       $title.text($chart.data('title'));
       $title.attr({
-        'text-anchor': 'start',
         x: legendBounds.left,
         y: legendBounds.top - 6,
         fill: '#000',
-        'font-family': 'Roboto, "Lucida Grande", Verdana, Arial, sans-serif',
+        'font-family': font,
         'font-weight': 'bold',
         'font-size': 13
       });
       $svg.append($title);
+
+      $svg.on('mouseover', function() {
```

Comment:
tablets? We don't have proper browser requirements for the admin features, but we should perhaps consider that some day a user will be looking at those numbers from their mobile device.